content/en/docs/setup/install/operator/snips.sh

```diff
@@ -24,14 +24,18 @@ snip_create_istio_operator() {
 istioctl operator init
 }
 
-snip_prerequisites_2() {
+snip_deploy_the_istio_operator_2() {
```

Comment:
The test looks good, but not seeing anywhere use the this func?

Reply:
this was addressed in 5d0ea40fd365c09ed4c42c04f5db56cf064efd9c - I have added additional tests and set `snip_id=none` for snippets that are not tested